sys/targets/targets.go

```diff
@@ -208,19 +208,19 @@ var List = map[string]map[string]*Target{
 		"amd64": {
 			PtrSize:           8,
 			PageSize:          4 << 10,
-			CCompiler:         "c++",
-			CFlags:            []string{"-m64"},
-			CrossCFlags:       []string{"-m64", "-static"},
+			CCompiler:         "clang",
```

Comment:
Ideally we use C compiler for C repros and C++ compiler for executor, right?
No need to do this change right now, just trying to understand the situation.

Reply:
You are absolutely right. Initially I use `c++` there, which works perfectly for compiling a C++ program. However, when compiling a C program with it, it does it, but warn that it is deprecated behaviour. I could have switched off the warning, but preferred  to switch to a C compiler. Then I had to add `-lc++`. This is also like a hack. It would be cleaner to provide a C compiler and a C++ compiler.